src/common/DamageTypes.js

```diff
@@ -0,0 +1,25 @@
+export const damageType = {
```

Comment:
Looking at logs and reverse engineered the list using the bits which are set in the damage type and the damage type listed on the tooltips on wowhead. Because i found instances of a few patterns of bits coming up as Chaos in tooltip i used that as a default and filled out the last 2 pairs from memory since i don't believe they are in game currently.

If the list is incomplete or wrong, I will update immediately.